antlr-kotlin-examples-jvm/settings.gradle

```diff
@@ -1,3 +0,0 @@
-rootProject.name = 'antlr-kotlin'
```

Comment:
why removing this?

Reply:
I thought this file is a copy of the settings.gradle of the root project. Maybe "rootProject.name = 'antlr-kotlin-examples-jvm'" whould be better? The build is working fine with the empty file.

Currently, I'm working on kotlin-dsl support for the gradle plugin. I will create/update example projects using plain gradle and another one using kotlin-dsl when I find some time for this.

Reply:
Right, thank you!
I am looking forward to the new patch!